packages/web/src/pages/audio-rewards-page/ChallengeRewardsTile.tsx

```diff
@@ -237,23 +267,49 @@ const ClaimAllPanel = () => {
         />
         <Flex direction='column'>
           <Flex>
-            <Text color='accent' size='m' variant='heading'>
-              {messages.totalReadyToClaim}
-            </Text>
-            <div className={wm(styles.pendingPillContainer)}>
-              <span className={styles.pillMessage}>
-                {pendingAmount} {messages.pending}
-              </span>
-            </div>
+            {claimableAmount > 0 ? (
+              <Text color='accent' size='m' variant='heading'>
```

Comment:
since these are the same text configs, lets do the ternery inside the text
so `<Text ...>{claimableAmount > 0 ? message1 : message2}</Text>`

Lets do the same for all the text in this file